src/compiler/checker.ts

```diff
@@ -4735,6 +4859,1104 @@ namespace ts {
+                        const commentText = jsdocAliasDecl ? jsdocAliasDecl.comment || jsdocAliasDecl.parent.comment : undefined;
+                        const oldFlags = context.flags;
+                        context.flags |= NodeBuilderFlags.InTypeAlias;
+                        addResult(setSyntheticLeadingComments(
+                            createTypeAliasDeclaration(/*decorators*/ undefined, /*modifiers*/ undefined, getInternalSymbolName(symbol, symbolName), typeParamDecls, typeToTypeNodeHelper(aliasType, context)),
+                            !commentText ? [] : [{ kind: SyntaxKind.MultiLineCommentTrivia, text: "*\n * " + commentText.replace(/\n/g, "\n * ") + "\n ", pos: -1, end: -1, hasTrailingNewLine: true }]
+                        ), modifierFlags);
+                        context.flags = oldFlags;
+                    }
+                    // Need to skip over export= symbols below - json source files get a single `Property` flagged
+                    // symbol of name `export=` which needs to be handled like an alias. It's not great, but it is what it is.
+                    if (symbol.flags & (SymbolFlags.BlockScopedVariable | SymbolFlags.FunctionScopedVariable | SymbolFlags.Property)
+                        && symbol.escapedName !== InternalSymbolName.ExportEquals
+                        && !(symbol.flags & SymbolFlags.Prototype)
+                        && !(symbol.flags & SymbolFlags.Class)) {
+                        if (propertyAsAlias) {
+                            serializeMaybeAliasAssignment(symbol);
+                        }
+                        else {
+                            const type = getTypeOfSymbol(symbol);
+                            const localName = getInternalSymbolName(symbol, symbolName);
+                            if (!(symbol.flags & SymbolFlags.Function) && isTypeRepresentableAsFunctionNamespaceMerge(type, symbol)) {
+                                // If the type looks like a function declaration + ns could represent it, and it's type is sourced locally, rewrite it into a function declaration + ns
+                                serializeAsFunctionNamespaceMerge(type, symbol, localName, modifierFlags);
+                            }
+                            else {
+                                // A Class + Property merge is made for a `module.exports.Member = class {}`, and it doesn't serialize well as either a class _or_ a property symbol - in fact, _it behaves like an alias!_
+                                // `var` is `FunctionScopedVariable`, `const` and `let` are `BlockScopedVariable`, and `module.exports.thing =` is `Property`
+                                const flags = !(symbol.flags & SymbolFlags.BlockScopedVariable) ? undefined : isConstVariable(symbol) ? NodeFlags.Const : NodeFlags.Let;
+                                const name = (needsPostExportDefault || !(symbol.flags & SymbolFlags.Property)) ? localName : getUnusedName(localName, symbol);

```

Comment:
The `localName` is the cached `getInternalSymbolName` which, since the symbol was named `default`, should be cached to `_default` (which is what we'll now refer to it as internally - hence why it's the "local name" of the symbol, and not its' exported name).